spring-cloud-dataflow-server-core/src/main/resources/schemas/db2/jpa.sql

```diff
@@ -1,13 +1,13 @@
     create sequence hibernate_sequence start with 1 increment by 1;
 
     create table APP_REGISTRATION (
-        id bigint not null,
-        object_version bigint,
-        default_version smallint,
-        metadata_uri varchar(255),
+       id bigint not null,
+        object_Version bigint,
```

Comment:
Are we changing the case here for any reason? I thought that the naming strategy would convert camelCase to underscore notation on db. mysql could barf on this much like it did with table names.